android/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,3 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.revenuecat.purchases_flutter">
```

Comment:
So this will be duplicated both here and in the namespace property right in the gradle file? I think it's ok though.

Reply:
Yes, I was worried AGP 8 would warn but it doesn't, so I think it's ok yes. I also noticed the official plugins have it duplicated too

https://github.com/flutter/packages/blob/main/packages/camera/camera_android/android/src/main/AndroidManifest.xml

https://github.com/flutter/packages/blob/main/packages/camera/camera_android/android/build.gradle#L31